js/build/shortcode-ui.js

```diff
@@ -721,6 +723,40 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 		this.$container.append( $thumbnail );
 		this.$container.toggleClass( 'has-attachment', true );
 
+		this.$metaContainer.empty();
+		var $thumbnailDetails = jQuery('<div class="thumbnail-details"></div>');
+
+		jQuery( '<strong/>', {
+			class: 'thumbnail-details-title',
+			html: 'ATTACHMENT DETAILS',
+		}).appendTo( $thumbnailDetails );
+
+		jQuery( '<div/>', {
+			class: 'filename',
+			html: attachment.filename,
+		}).appendTo( $thumbnailDetails );
+
+		jQuery( '<div/>', {
+			class: 'date-formatted',
+			html: attachment.dateFormatted,
+		}).appendTo( $thumbnailDetails );
+
+		jQuery( '<div/>', {
+			class: 'size',
+			html: attachment.filesizeHumanReadable ,
+		}).appendTo( $thumbnailDetails );
+
+		jQuery( '<div/>', {
+			class: 'dimensions',
+			html: attachment.height + ' &times; ' + attachment.width ,
+		}).appendTo( $thumbnailDetails );
+
+		jQuery( '<div/>', {
+			class: 'edit-link',
+			html: '<a href="' + attachment.editLink + '">Edit Image</a>',
+		}).appendTo( $thumbnailDetails );
+
+		this.$metaContainer.append( $thumbnailDetails );
```

Comment:
I'm not a big fan of all this jQuery DOM-building going on here in the view. Seems like this is what templates are for. 

Is there a way to rework this field so that it makes better use of templates, both here and in the image preview?